build-cmd.sh

```diff
@@ -209,20 +203,23 @@ pushd "$PNG_SOURCE_DIR"
             # * Builds all bin/* targets with static libraries.
 
             # build the release version and link against the release zlib
+
             CFLAGS="$opts" \
                 CXXFLAGS="$opts" \
                 CPPFLAGS="${CPPFLAGS:-} -I$stage/packages/include/zlib" \
                 LDFLAGS="-L$stage/packages/lib/release" \
+
+            #LL_BUILD_RELEASE contains a c++ standard flag which are meaningful to the c++
+            #compiler only (g++), when introduced into CFLAGS it causes gcc (the C compiler)
+            #to spam a load of noise.. Strip out any C++ std cruft from CFLAGS (squelches the noise)
+            CFLAGS=$(echo "$CFLAGS" | sed 's/-std=c++[0-9][0-9]*//')
+
```

Comment:
I suggest inserting at line 30 or thereabouts:

```
# remove_cxxstd
source "$(dirname "$AUTOBUILD_VARIABLES_FILE")/functions"
```

Then at lines 126 and 207:

```
CFLAGS="$(remove_cxxstd $opts)" \
```

That's what we've done in some other 3p `build-cmd.sh` scripts.